catalog.html

```diff
@@ -0,0 +1,160 @@
+        <li class="main-menu-item"><a class="main-menu-link" href="#"></a>Контакты></li>
+      </ul>
+      <section class="next-menu">
+        <form class="search">
+          <input type="text" value="Текст поиска">
+          <button type="submit">Найти</button>
+        </form>
+        <ul class="button-menu">
+        <li class="button-menu-icon"><a href="Коля Петухов"></a></li>
+        <li class="button-menu-icon"><a href="#" class="button">Войти</a></li>
+        <li class="button-menu-icon"><a href="#" class="button">Выйти</a></li>
+        <li class="button-menu-icon"><a href="#" class="button">Сравнить</a></li>
+        <li class="button-menu-icon"><a href="#" class="button">Корзина</a></li>
+        </ul>
+      </section>
+    </nav>
+    </header>
+    <main class="main-catalog">
+    <h1 class="main-catalog-pods">Моноподы для селфи</h1>
+    <ul class="main-menu-catalog">
+      <li class="menu-catalog-navigation"><a href="#" class="button">Главная</li>
+      <li class="menu-catalog-navigation"><a href="#" class="button">Каталог товаров</li>
+      <li class="menu-catalog-navigation"><a href="#" class="button">Моноподы для селфи</li>
+      <section class="filter">
+      <form action="https://echo.htmlacademy.ru">
+          <fieldset class="catalog-filter-group">
+            <legend class="catalog-filter-title">Стоимость,₽</legend>
+            от
+          <label><input class="conteol-input-min" type="number" name="min-price" value="0"></label>
+            до
+          <label><input class="conteol-input-min" type="number" name="max-price" value="5000"></label>
+          </fieldset>
+
+
+          <fieldset class="catalog-filter-color">
+            <legend class="catalog-filter-color">Цвет</legend>
+          <div>
+            <input type="checkbox" id="black" name="black" value="black" checked>
+            <label for="black">Черный</label>
+          </div>
+          <div>
+            <input type="checkbox" id="white" name="white" value="white" checked>
+            <label for="white">Белый</label>
+          </div>
+          <div>
+            <input type="checkbox" id="blue" name="blue" value="blue">
+            <label for="blue">Синий</label>
+          </div>
+          <div>
+            <input type="checkbox" id="red" name="red" value="red">
+            <label for="red">Красный</label>
+          </div>
+          <div>
+            <input type="checkbox" id="pink" name="pink value="pink">
+            <label for="pink">Розовый</label>
+          </div>
+        </fieldset>
+        <fieldset class="catalog-filter-radio">
+        <legend class="catalog-filter-radio">bluetooth</legend>
+          <div>
+            <input type="radio" id="yes" name="bluetooth" value="Есть" checked>
+            <label for="yes">Есть</label>
+          </div>
+          <input type="radio" id="no" name="bluetooth" value="Нет">
+            <label for="no">Нет</label>
+            <a href="#" class="button">Подписаться</a>
```

Comment:
Что-то тут напутал ты.
```suggestion
             <label for="no">Нет</label>
```